benchmarks/operator_benchmark/benchmark_runner.py

```diff
@@ -73,13 +73,14 @@ def main():
 
     parser.add_argument(
         '--framework',
-        help='Run PyTorch or Caffe2 operators',
-        default=None)
+        help='Comma-delimited list of frameworks to test (Caffe2, PyTorch)',
+        default="Caffe2,PyTorch")
```

Comment:
I think you can put `action=append` here and not rely on comma-separated list

Reply:
not sure which one you mean, might be appendTuple, though comma-separated list of also pretty standard way to pass args

Reply:
though in this case we'll have to use `--framework` multiple times, I think comma-separated list is more standard and compact